### DISCUSSION TITLE
27-03-2024 Steering Committee Meeting Minutes

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#120] [Steering Committee]
1. Discuss an new website/email solutions, considering using web2+web3 name servicing dcdao.box as new official website after skiff being acquired by Notion, https://github.com/orgs/dcdao/discussions/118, an issue for email DMIK settings is pending for the feedback of my.box, @casinoyoda is tracking on this.
2. Requesting monthly reports from managers cc @2xJDubs @PartnershipDude @nblogist 
3. Review/discuss new proposals on https://snapshot.org/#/darwiniacommunitydao.eth

**Comment by nblogist** [comment]
Hi @hackfisher I made my [last report](https://github.com/orgs/dcdao/discussions/101) from the 15th to the end of the month and will be making future reports on each month end, Next report due after the 31st of March 2024.

**Comment by 2xJDubs** [comment]
@hackfisher like nblogist I last submited a monthly report for the 15th of March but am happy to move to the beginning of each new month.  I will have my next Community Manager report up by the 3rd of April.
